kanidm_book/src/accounts_and_groups.md

```diff
@@ -27,25 +27,40 @@ and sensitive data), group management, and more.
 ## Recovering the Initial idm_admin Account
 
 By default the idm_admin user has no password, and can not be accessed. You should recover it with the
-admin (system admin) account. We recommend the use of "reset_credential" as it provides a high
-strength, random, machine only password.
+admin (system admin) account. We recommend the use of the "recover_account" functionalit as it provides a high strength, random password.
 
-    kanidm account credential reset_credential  --name admin idm_admin
-    Generated password for idm_admin: tqoReZfz....
+<table>
+<tr>
+<td>
+<img src="/images/kani-warning.png" style="float:left">
+</td>
+<td>Warning: The server must not be running at this point, as it requires raw access to the database.</td>
```

Comment:
Maybe "full access" to the database is a better framing than raw. Also point out that this is why we don't show the "docker exec" version of recovery.

Reply:
I've added a howto on docker, updated the wording a bit.

Reply:
Good idea. :) Another option is instead of making it a "server" binary command like this, have a unix domain socket or similar on localhost to allow the docker exec pattern to work?

Reply:
That feels like a great way to make it easy to get root on everyone's box. 😄  Force-recovering an account *should* be a break-glass move with *really* obvious outcomes.

IMO, doing it on *any* account should have some pretty wide-ranging effects, like wiping sessions domain-wide and ... being a real jerk about force-replicating it or something like that - it should be a big-boom effect similar to granting a new golden ticket on an AD Domain.

Reply:
well to run the command you'd need root on the IDM machine anyway which is already game over .... 

But yes, it should be a break glass move. Anyway, I think improvements to db locking will help.